modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/journal/structures/img_Hero.json

```diff
@@ -0,0 +1,32 @@
+{
+    "availableLanguageIds": [
+        "en_US"
+    ],
+    "defaultLanguageId": "en_US",
+    "fields": [
+        {
+            "label": {
+                "en_US": "img_Hero"
+            },
+            "predefinedValue": {
+                "en_US": ""
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": "Upload your front-page hero image."
+            },
+            "dataType": "image",
+            "fieldNamespace": "ddm",
+            "indexType": "keyword",
+            "localizable": true,
+            "name": "img_Hero",
```

Comment:
Field name should be all lower case since we have the underscore to separate the words.

Ex.
```
hero_image
```